src/components/LoginForm/LoginOptions.tsx

```diff
@@ -0,0 +1,38 @@
+import '../../i18n/i18n.ts';
+import { Checkbox, FormControlLabel, Grid, Link } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+export function LoginOptions() {
+  const { t } = useTranslation();
+  return (
+    <Grid
+      container
+      direction={{
+        desktop: 'row',
+        mobile: 'column',
+        tablet: 'row',
+      }}
```

Comment:
This is not needed.
you can simplify check to change layout if we reach table+ resolution:
```tsx
const isTablet = useMediaQuery(theme.breakpoints.up('tablet'));

return (
    <Grid
      container
      direction={isTablet ? 'row' : 'column'}

```